model-engine/model_engine_server/inference/configs/service--http_forwarder.yaml

```diff
@@ -9,6 +9,7 @@ forwarder:
     model_engine_unwrap: true
     serialize_results_as_string: true
     forward_http_status: true
+    extra_routes: []
```

Comment:
btw it might be hard to define behavior from here, we may need to end up passing this as some runtime parameter to the http forwarder (e.g. through cli args)

Reply:
yeah i added this here to illustrate that the field exists. We'd probably need to do something similar to how we pass in the predict route as env var at model bundle creation time